docs/topics/wf-swarm-runtime-overview.adoc

```diff
@@ -0,0 +1,6 @@
+= Runtime Details
+//http://wildfly-swarm.io
+
+{WildFlySwarm} deconstructs the features in link:http://wildfly.org[WildFly Java Application Server] and allows them to be selectively reconstructed based on the needs of your application. This allows you to create microservices that run on a _just-enough-appserver_ that supports the exact subset of JavaEE APIs you need. Check out xref:wf-swarm-additional-resources[Additional Resources] for further reading on {WildFlySwarm}.
```

Comment:
"WildFly Java Application Server" should probably read "WildFly Java EE Application Server"?